### ISSUE TITLE
Game starting logged in restart and it's not but its not or it isnt entirely?

### ISSUE THREAD

**Issue by DoWeBeepBoop** [Yellow-Dog-Man/Resonite-Issues#2725]
### Describe the bug?

For the past week and a half I've been having an issue where I'd login selecting the keep signed in 30 days it would launch in fine every thing loads fine come the day after it will launch look fine but no contacts I restart and i'm signed out. I've wiped the game cache and all install directories and reinstall the issue still remains. And while trying to make this recreate it has the same behavior that it has with the issue but from a fresh login.

### To Reproduce

Login run than restart the game..

### Expected behavior

Just be logged in and contacts show.

### Screenshots

![Screenshot 2024-08-06 141907](https://github.com/user-attachments/assets/bc43e0e1-96b1-4837-a0de-29d8d6d16654)
![Screenshot 2024-08-06 141915](https://github.com/user-attachments/assets/4d513d93-c913-4ffd-ac04-b0b51a6859df)
![Screenshot 2024-08-06 142038](https://github.com/user-attachments/assets/a1d9e4bd-64ce-489f-8be0-cd937de5ca36)

### Resonite Version Number

2024.8.5.1341

### What Platforms does this occur on?

Windows

### What headset if any do you use?

Vive

### Log Files

[THEBOXOFBOLTS - 2024.8.5.1341 - 2024-08-06 14_08_49.log](https://github.com/user-attachments/files/16505219/THEBOXOFBOLTS.-.2024.8.5.1341.-.2024-08-06.14_08_49.log)
[THEBOXOFBOLTS - 2024.8.5.1341 - 2024-08-06 14_06_47.log](https://github.com/user-attachments/files/16505221/THEBOXOFBOLTS.-.2024.8.5.1341.-.2024-08-06.14_06_47.log)
[THEBOXOFBOLTS - 2024.8.5.1341 - 2024-08-06 14_19_29.log](https://github.com/user-attachments/files/16505222/THEBOXOFBOLTS.-.2024.8.5.1341.-.2024-08-06.14_19_29.log)
[THEBOXOFBOLTS - 2024.8.5.1341 - 2024-08-06 14_16_28.log](https://github.com/user-attachments/files/16505223/THEBOXOFBOLTS.-.2024.8.5.1341.-.2024-08-06.14_16_28.log)

### Reporters

IGN: Near By Gamer / Discord: nearbygamer

**Comment by shiftyscales** [comment]
> I've wiped the game cache and all install directories

Which folders in specific are you referring to here, @DoWeBeepBoop?

Does the issue persist if you entirely delete the database path as well as the cache path?

https://wiki.resonite.com/Resonite_Data_Folders

**Comment by shiftyscales** [comment]
Optionally, you could instead use command line arguments to set new directories for them, and see if the issue occurs on a new database that way. https://wiki.resonite.com/Relocate_Resonite_Folders

**Comment by DoWeBeepBoop** [comment]
I deleted everything under the user folder that connects with resonite

**Comment by DoWeBeepBoop** [comment]
Here's what's going on 

Uploading 2024-08-16 20-38-27.H264.mp4…

**Comment by DoWeBeepBoop** [comment]
Here whats going on

https://github.com/user-attachments/assets/7ddcc4e9-63b3-4fc6-a968-31caa22f639d

**Comment by shiftyscales** [comment]
I'd recommend you try to reset your dash and see if the issue persists, @DoWeBeepBoop. You can do this with the `-ResetDash` command line argument. Details on this command, and how to use it are on our wiki: https://wiki.resonite.com/index.php?title=Command_Line_Arguments

**Comment by DoWeBeepBoop** [comment]
looks likes it's a conflict with IPv6 the fix recomended didn't fix

**Comment by ProbablePrime** [comment]
Spoke to the user about this for awhile earlier.

They've stopped experiencing it, and I think it lines up with some information in: https://github.com/Yellow-Dog-Man/Resonite-Issues/issues/1924 and #2239 and 2024.6.5.198

I don't think this is related to IPV6.

However as the user has confirmed this issue has not re-occured closing.

**Comment by ProbablePrime** [comment]
Had a longer chat a few weeks ago with this user and I do actually think its IPV6. They are still intermittently experiencing it, so re-opening.

Also providing two log files from them.
[THEBOXOFBOLTS - 2024.8.5.1341 - 2024-09-03 19_22_59.log](https://github.com/user-attachments/files/17143970/THEBOXOFBOLTS.-.2024.8.5.1341.-.2024-09-03.19_22_59.log)
[THEBOXOFBOLTS - 2024.8.5.1341 - 2024-09-03 19_25_43.log](https://github.com/user-attachments/files/17143971/THEBOXOFBOLTS.-.2024.8.5.1341.-.2024-09-03.19_25_43.log)

I have no idea if this is what's happening, but perhaps there's some session token generation/validation stuff that doesn't work due to IPV6 for some reason which is causing their token to be somewhat broken. That would explain them being logged out and contacts(signalr) not accepting their identity.

**Comment by bredo228** [comment]
Encountered this issue earlier - in troubleshooting, I noted that api.resonite.com would not respond to ping over IPv6 but would respond over IPv4.

Another computer on the same network was able to ping (and access) api.resonite.com over IPv6 at the same time as the computer having issues was failing to ping over IPv6. Restarting the computer having issues resolved the problem

[ERIS - 2024.12.18.477 - 2024-12-27 12_39_51.log](https://github.com/user-attachments/files/18256200/ERIS.-.2024.12.18.477.-.2024-12-27.12_39_51.log)
[ERIS - 2024.12.18.477 - 2024-12-27 12_41_57.log](https://github.com/user-attachments/files/18256201/ERIS.-.2024.12.18.477.-.2024-12-27.12_41_57.log)

**Comment by troyBORG** [comment]
Yeah, I had the same issue a few days ago as well. I ended up having to log into my router and tell it to release and renew the IP address and then that seemed to fix it where it would then like log in normally.  

I'm on my phone right now, but let me go over to my PC and I will see about getting you the log files.
== Log files will be posted here.==
